frontend-react/src/components/header/ReportStreamNavbar.tsx

```diff
@@ -82,25 +92,63 @@ export const ReportStreamNavbar = ({
     };
     const defaultMenuItems = [
         <div className="primary-nav-link-container">
-            <a className="primary-nav-link" href="/product" key="product">
-                Product
+            <a
+                className={primaryLinkClasses(!!useMatch("/product/*"))}
+                href="/product/overview"
+                key="product"
+            >
+                About
             </a>
         </div>,
         <div className="primary-nav-link-container">
-            <a className="primary-nav-link" href="/resources" key="resources">
-                Resources
+            <a
+                className={primaryLinkClasses(!!useMatch("/getting-started/*"))}
+                href="/getting-started"
+                key="resources"
+            >
+                Getting started
             </a>
         </div>,
         <div className="primary-nav-link-container">
-            <a className="primary-nav-link" href="/support" key="support">
+            <a
+                className={primaryLinkClasses(
+                    !!useMatch("/developer-resources/*"),
+                )}
+                href="/developer-resources"
+                key="support"
+            >
+                Developers
+            </a>
+        </div>,
+        <div className="primary-nav-link-container">
+            <a
+                className={primaryLinkClasses(
+                    !!useMatch("/managing-your-connection/*"),
```

Comment:
@penny-lischer confirming that this is the right link?

Reply:
It is not, should be "manage-connection" but I suggest you move the index file from "manage-connection" into "managing-your-connection" and then delete the "managing-connection" folder. 
Joseph and I had some overlap.

Reply:
fixed!